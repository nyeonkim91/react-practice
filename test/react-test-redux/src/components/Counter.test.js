import React from 'react';
import { shallow } from 'enzyme';
import Counter from './Counter';

describe('Counter', () => {
  let component = null;

  it('renders correctly', () => {
    component = shallow(<Counter />);
  });

  it('matches snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
