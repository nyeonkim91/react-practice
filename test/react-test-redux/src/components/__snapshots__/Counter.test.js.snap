// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter matches snapshot 1`] = `
<div>
  <h1>
    카운터
  </h1>
  <h2 />
  <button>
    +
  </button>
  <button>
    -
  </button>
</div>
`;
